src/blocks/utils/generated/external_samples_data.json

```diff
@@ -698,7 +712,21 @@
             ],
             "instanceVariables": [],
             "moduleName": "rev_touch_sensor",
-            "staticMethods": []
+            "staticMethods": [
+                {
+                    "args": [
+                        {
+                            "defaultValue": null,
+                            "name": "io_port",
+                            "type": "int"
+                        }
+                    ],
+                    "declaringClassName": "rev_touch_sensor.RevTouchSensor",
```

Comment:
Since this is in JSON that already has className, why does it have to be repeated?

Reply:
I'm not sure. Maybe when I wrote the python code that generates the JSON I thought there could be a difference between a function that is inherited from a base class? I'll have to look into that.
I filed https://github.com/alan412/mrc_python_tools/issues/14